scripts/commands/spin.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- func: spin <player>
+-- desc: Spins a character in place. Used for fishbot detection.
```

Comment:
Could you expand the comment here and explain how this is used to observe/trap/detect fishbotters?

Reply:
Suppose to turn them in different direction to help catch if doing any fishbot.